INSTALL.md

```diff
@@ -52,8 +42,7 @@ make -j
 sudo make install
```

Comment:
I'd suggest `-DLLVM_ENABLE_PROJECTS="clang;lld"` instead of `-DLLVM_EXTERNAL_LLD_SOURCE_DIR=../lld -DLLVM_EXTERNAL_CLANG_SOURCE_DIR=../clang`